recipe/test-rpath.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
+for lib in `find ${PREFIX}/${targetsDir}/lib -type f`; do
+    [[ $lib =~ \.so ]] || continue
+
+    rpath=$(patchelf --print-rpath $lib)
+    echo "$lib rpath: $rpath"
+    [[ $rpath == "\$ORIGIN" ]] || exit 1
```

Comment:
Maybe track if there were any failures in a variable. Then you can print the results for all the libs first and `exit 1` after the loop if there were any problematic libraries.

Reply:
Added here https://github.com/conda-forge/cuda-nvtx-feedstock/pull/5/commits/9eb076121dd746c03e67a681648eba8df2044837